demos/python_demos/multi_camera_multi_person_tracking/mc_tracker/mct.py

```diff
@@ -23,18 +24,27 @@ class MultiCameraTracker:
     def __init__(self, num_sources, reid_model,
                  sct_config={},
                  time_window=20,
-                 global_match_thresh=0.35
+                 global_match_thresh=0.35,
+                 bbox_min_aspect_ratio=1.2,
+                 visual_analyze=None,
+                 random_seed=100
                  ):
         self.scts = []
         self.time = 0
         self.last_global_id = 0
         self.global_ids_queue = queue.Queue()
+        assert time_window >= 1
         self.time_window = time_window  # should be greater than time window in scts
+        assert 0 <= global_match_thresh <= 1
         self.global_match_thresh = global_match_thresh
+        assert bbox_min_aspect_ratio >= 0
+        self.bbox_min_aspect_ratio = bbox_min_aspect_ratio
+        random.seed(random_seed)
```

Comment:
This seems misleading. If `MultiCameraTracker` accepts a `random_seed` parameter, then it should only use that seed for its own calculations, not set it globally. If you want to set a seed globally, just do it in `main`. You don't need to pass it as a parameter.

Reply:
`random_seed` setup moved to main function.